### PR TITLE
No bug - Fix possible XSS bug in the tags display

### PR DESCRIPTION
**Pull request by jotes**
Hey,
As We discussed, this should fix the possible XSS attack from anyone who's able to create new tags.

I decided to escape only  `slug` and `name` fields because they have `CharField` type.
//cc @mathjazz

### CODE REVIEW

File: pontoon/base/static/js/translate.js [hunk 1]
```diff
@@ -3215,15 +3215,16 @@ var Pontoon = (function (my) {
 
       tags.forEach(function(tag) {
         var priority = '';
+        var slug = Pontoon.doNotRender(tag.slug);
```
Comment (mathjazz) [inline pontoon/base/static/js/translate.js]: I'd add `var name = Pontoon.doNotRender(tag.name);` here and a comment on top of these two lines explaining why is that needed. Other than that, this is good to go. :)

File: pontoon/base/static/js/translate.js [hunk 2]
```diff
@@ -3216,14 +3216,19 @@ var Pontoon = (function (my) {
       tags.forEach(function(tag) {
         var priority = '';
 
+        // In order to avoid possible XSS attacks these variables can't be rendered as HTML,
```
Comment (jotes) [inline pontoon/base/static/js/translate.js]: @mathjazz could you check if this comment is okay?
